worker/firewaller/firewaller.go

```diff
@@ -270,6 +287,14 @@ func (fw *Firewaller) setUp() error {
 		return errors.Trace(err)
 	}
 
+	fw.modelConfigWatcher, err = fw.firewallerApi.WatchForModelConfigChanges()
```

Comment:
This should not be exposed to the worker as a model config watcher. The fact that we use model config to store this stuff is an implementation artefact. The api caller side of the watcher should expose a method like "WatchModelModelwallRules" or something. Internally it can call model config watcher, but that shouldn't be exposed.